### DISCUSSION TITLE
ui suggestion - add send icon?

### DISCUSSION THREAD

**Discussion by jasonbrown1965** [localsend/localsend#1275] [Discussion (English, International)]
In absolute awe at TotalSend - worked out of the box with my weird ethernet setup, despite the install checklist warning about being on the same "wifi" network - cable works fab too!

Suggestion, tho?

As much as there is to love, love, love the elegant simplicity of the app interface? It lacks a certain obviousness when it comes time to send. Must have spent a few minutes looking for the usual kind of icon eg an arrow shape > or a button titled "send". Only when I clicked on the host tab (?section) to see what might be wrong - oh! The entire tab is a send button.

Writing this, I'm vaguely remembering articles about an emerging design ethos, where every element is actionable, and that's great, but in an icon-driven world, suggest adding an icon to match the overall simplicity of design.

Please pardon my MS paint skills, but something like this?

![image](https://github.com/localsend/localsend/assets/11400799/4f9e2705-f880-41a0-85a7-a4fe9a51efb9)

Otherwise, from a non-tech user perspective, flawless. 

Edit: found Localsend after trying syncthing, warpinator and sparkleshare - none worked out of the box

**Comment by Tienisto** [comment]
Hi, thanks for your feedback.
I just don't know how to add the send icon when there is already a favorite icon
